lib/xlsx/xform/style/alignment-xform.js

```diff
@@ -56,10 +56,10 @@ var validation = {
     return Math.max(0, value);
   },
   readingOrderValues: [
-    {k: 'r2l', v: 1},
-    {k: 'l2r', v: 2},
-    {k: Enums.ReadingOrder.RightToLeft, v: 1},
-    {k: Enums.ReadingOrder.LeftToRight, v: 2}
+    {k: 'rtl', v: 2},
```

Comment:
Would you be able to add a unit test for these changes?
in spec/unit/xlsx/xform/style/alignment-xform.spec.js